Mathlib/NumberTheory/Cyclotomic/Discriminant.lean

```diff
@@ -153,7 +153,7 @@ theorem discr_prime_pow [hcycl : IsCyclotomicExtension {p ^ k} K L] [hp : Fact (
       minpoly.eq_X_sub_C_of_algebraMap_inj _ (algebraMap K L).injective, natDegree_X_sub_C]
     simp only [traceMatrix, map_one, one_pow, Matrix.det_unique, traceForm_apply, mul_one]
     rw [← (algebraMap K L).map_one, trace_algebraMap, finrank _ hirr]
-    simp; norm_num
+    simp
```

Comment:
Could you please fix this non-terminal `simp`? Either use `simp only` or `suffices ... by simpa`

Reply:
All is well: this is a already terminal simp!

Reply:
Is this related to the addition of the new parity simp lemmas? I so, does
```suggestion
    norm_num
```
by itself work, as I believe that calls `simp`.

Reply:
Yes, the two changes to this file are necessary because of the `simp`-lemma tinkering. I've changed this to `norm_num`.